lib/src/app.dart

```diff
@@ -38,7 +38,7 @@ class AppConfiguration {
   /// directory exists.
   final Directory baseFilePath;
 
-  /// The [baseUrl] is the [Uri] used to reach the MongoDB Realm server.
+  /// The [baseUrl] is the [Uri] used to reach the MongoDB Atlas.
```

Comment:
```suggestion
  /// The [baseUrl] is the [Uri] used to reach Atlas App Services.
```

Technically, those are different urls today - MongoDB Atlas is at `cloud.mongodb.com` and AAS is at `realm.mongodb.com`. I imagine the second will change, but stay different. So users shouldn't type in the Atlas url as the `baseUrl`.